crdppf/views/printproxy.py

```diff
@@ -87,17 +100,45 @@ def report_create(self):
         )
 
         body = json.dumps(body)
-
+        
         # Specify correct content type in headers
         h = dict(self.request.headers)
         h["Content-Type"] = "application/json"
 
-        return self._proxy_response(
+        print_result = self._proxy_response(
             _string,
             body=body,
             method='POST',
             headers=h
         )
+          
+        if len(pdfs_to_join) > 0:
+            main = tempfile.NamedTemporaryFile(suffix='.pdf')
+            main.file.write(print_result.body)
+            main.flush()
+            cmd = ['pdftk', main.name]
+            temp_files = [main]
+            for url in pdfs_to_join:
+                tmp_file = tempfile.NamedTemporaryFile(suffix='.pdf')
+                result = requests.get(url)
+                tmp_file.write(result.content)
+                #~ pdf = urlopen(Request(url)).read()
+                #~ tmp_file.write(pdf)
+                tmp_file.flush()
+                temp_files.append(tmp_file)
+                cmd.append(tmp_file.name)
+            out = tempfile.NamedTemporaryFile(suffix='.pdf')
+            cmd += ['cat', 'output', out.name]

```

Comment:
What is this command `cat` ?? Is it some kind of PDF aggregator ?

If yes, you should describe it as a mandatory soft to install + define in the path in the requirements...